src/pe/mod.rs

```diff
@@ -176,6 +176,26 @@ impl<'a> PE<'a> {
             exception_data,
         })
     }
+
+    /// Validate the `PE`
+    pub fn validate(&self) -> error::Result<()> {
```

Comment:
This isn't turned on by default, right? I.e., it's a method the user can (optionally) call after to do validation after the parse occurs, and PE object created (assuming it didn't fail from more serious errors during initial parse), yes?